### PR TITLE
[R4R]: Add trade section

### PR DESCRIPTION
**Pull request by fabryscript**
⚠️ WARNING FOR REVIEWERS ⚠️

We still don't have definitive copies for the texts in the cards and/or in some sections, please be aware of this differences and review accordingly 🙏🏻

### CODE REVIEW

File: components/sections/trade-section/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          <div className="flex flex-col gap-4 self-stretch md:self-start">
+            <h4 className="inline-flex flex-col gap-1.5 font-h1 text-3.5xl font-medium leading-9 -tracking-[0.64] text-neutral-100 sm:flex-row md:flex-col md:gap-0 md:text-4xl md:-tracking-[0.72%] lg:text-5xl lg:leading-[53px] lg:-tracking-[0.96px] xl:text-[56px] xl:leading-[61px] xl:-tracking-[1.12px] 2xl:text-[64px] 2xl:leading-[70px] 2xl:-tracking-[1.28px]">
+              <span className="bg-trade-headline">Start trading</span> today.
+            </h4>
+            <p className="self-stretch leading-[26px] text-neutral-100 md:w-[280px] lg:w-[368px] 2xl:w-[475px]">
+              Simply set up your wallet and connect to Osmosis instantly, from
+              anywhere. No sign-ups required.
+            </p>
+          </div>
+          <button
```
Comment (DavideSegullo) [inline components/sections/trade-section/index.tsx:17]: Why don't we create a single component for all the buttons? We can reuse some code here

**Review by DavideSegullo — APPROVED** [review]
Do not forget to use a single svg for the BG

File: components/sections/trade-section/index.tsx (hunk 1 above)
Comment (fabryscript) [reply components/sections/trade-section/index.tsx:17]: part of #10

**Comment by fabryscript** [comment]
> Do not forget to use a single svg for the BG

ref in #31

**Pull request merged**